cfgrammar/src/lib/newlinecache.rs

```diff
@@ -84,9 +89,12 @@ impl NewlineCache {
             .and_then(|line_num| self.line_num_to_byte(line_num))
     }
 
-    /// Given a `src` string, which should have strings previously passed to `feed` concatenated together.
-    /// Convert the given `byte` offset in the input to logical line and column numbers.
-    /// Returns None if the byte offset exceeds the known input length, or the src and input length differ.
+    /// A convenience method to return the logical line and logical column number of a byte. This
+    /// requires passing a `&str` which *must* be equivalent to the string(s) passed to `feed`.
+    //
+    /// # Panics
+    ///
```

Comment:
We can't *guarantee* to panic if they pass in an incorrect string but we *might* panic (e.g. because we try to index in the middle of a unicode char).

Sort-of clarified in c9731d1.